layouts/partials/engineering-education/trending-articles.html

```diff
@@ -96,9 +95,7 @@ <h2 class="title-3 xs-mb-20">Trending Articles</h2>
             </tr>
           </thead>
           <tbody id="trendingAllTimeTableBody" class="table-body">
-            {{ $dataTrendingAllTime := getJSON (print $trendingAllTimeUrl) }}
-
-            {{ with $dataTrendingAllTime }}
+            {{ with getJSON (print $trendingAllTimeUrl) }}
```

Comment:
I'm just curious, how do these `with` blocks help handle errors?

Reply:
Well this is a slight refactor, but also by adding the `else` blocks we can render the text that there were errors in the request.